CONTRIBUTING.rst

```diff
@@ -35,37 +43,47 @@ No local Postgres server needed.
 
 In the virtual environment you need to install *aiopg* itself and some
 additional development tools (the development tools are needed for running
-the test suite and other development tasks)::
+the test suite and other development tasks)
+
+.. code-block:: shell
 
-   $ pip install -Ue .
-   $ pip install -Ur requirements.txt
+    $ pip install -Ue .
+    $ pip install -Ur requirements.txt
 
 That's all.
 
-To run all of the *aiopg* tests do::
+To run all of the *aiopg* tests do
 
-   $ make test
+.. code-block:: shell
+
+    $ make test
 
 This command runs :term:`pep8` and :term:`pyflakes` first and then executes
 the *aiopg* unit tests.
 
 
 When you are working on solving an issue you will probably want to run
-some specific test, not the whole suite::
+some specific test, not the whole suite
+
+.. code-block:: shell
 
-   $ py.test -s -k test_initial_empty
+    $ py.test -s -k test_initial_empty
 
 For debug sessions I prefer to use :term:`ipdb`, which is installed
-as part of the development tools.  Insert the following line into your
```

Comment:
Some style guides (like PEP 257) actually require double space after dots. But it's up to you here...